include/operations/blas3_trees.h

```diff
@@ -42,7 +42,12 @@ enum class gemm_memory_t : int { local = 0, no_local = 1 };
  * It can be either naive to use a naive algorithm, classic for the default
  * algorithms, or tall_skinny for tall and skinny matrices
  */
-enum class gemm_algorithm_t : int { naive = 0, classic = 1, tall_skinny = 2 };
+enum class gemm_algorithm_t : int {
+  naive = 0,
+  classic = 1,
+  tall_skinny = 2,
+  alternative = 3
```

Comment:
what does alternative means?
do you call normal gemm (non-skinny) classic? classic gemm algorithm is the name chosen to call the 3 nested for loop doing the matmul. when you call the optimize gemm for non-skinny classic, it can be confusing

Reply:
The 3 nested loops is the `naive` algorithm. Anyway, as answered to Ross I can change that but I can't think of better names, so do you have something in mind?

Reply:
why do you need separate name. you tsgem is a gemm with numkgroup==1. and you should use the same gemm. you dont need to name it separately

Reply:
As discussed offline, `alternative` has been integrated in `tall_skinny` with simple heuristics to make it final. `classic` has been renamed to `standard`